pyproject.toml

```diff
@@ -80,7 +74,7 @@ doc = [
     "mne-nirs @ https://github.com/mne-tools/mne-nirs/archive/refs/heads/main.zip",
     "seaborn",
     "openneuro-py",
-    "defusedxml",  # for reading BrainVision montages: `examples/convert_eeg_to_bids.py`
+    "defusedxml",                                                                   # for reading BrainVision montages: `examples/convert_eeg_to_bids.py`
```

Comment:
is this is mistake or by design? if by design, why?

Reply:
Btw the linter I use comes with the Even Better TOML extension for VS Code

Very helpful tool really